build-tools/automation/azure-pipelines-oss.yaml

```diff
@@ -16,9 +16,6 @@ pr:
 # Global variables
 # Predefined variables: https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml
 variables:
```

Comment:
Consider adding a comment that references the build definition variable settings:
https://dev.azure.com/xamarin/public/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=48&view=Tab_Variables